debian/control

```diff
@@ -0,0 +1,33 @@
+Source: argobots
+Section: libs
+Priority: optional
+Maintainer: daos-stack <daos@daos.groups.io>
+Build-Depends: debhelper (>=10)
+Standards-Version: 4.1.4
+Homepage: https://wiki.hpdd.intel.com/display/DC/DAOS+Community+Home
+Vcs-Git: https://github.com/pmodels/argobots.git
+Vcs-Browser: https://github.com/pmodels/argobots
+
+Package: argobots1
```

Comment:
Why is the appended version 1 and not 0 given that the library (and presumably the SONAME) is _libabt.so.0.0.0_ (i.e. version 0)?

https://www.debian.org/doc/debian-policy/ch-sharedlibs.html#run-time-shared-libraries:

Normally, the run-time shared library and its SONAME symlink should be placed in a package named libraryname*soversion*, where soversion is the version number in the SONAME of the shared library. Alternatively, if it would be confusing to directly append soversion to libraryname (if, for example, libraryname itself ends in a number), you should use libraryname-soversion instead.

There is an interesting [link to a bit of code](https://www.debian.org/doc/debian-policy/ch-sharedlibs.html#id7) to automatically generate that name

Reply:
The Debian packaging system really wants this to be named libabt# and libabt-dev.
I tried setting it to argobots0 and the debian packaging had all sorts of issues with that name.

I can try it again.

Reply:
Renamed the packages to libabt0 and libabt-dev for debian.